src/clj/clojure/lang/persistent_set.clj

```diff
@@ -1,21 +1,21 @@
 (ns clojure.lang.persistent-set
-  (:refer-clojure :only [apply cons defn every? fn])
+  (:refer-clojure :only [apply defn fn])
   (:require [clojure.lang.protocols :refer [-conj -difference -disj -intersection -union]]
-            [clojure.next           :refer :all :exclude [cons every? conj]]))
+            [clojure.next           :refer :all]))
 
 (defn difference
   ([this] this)
-  ([this s] (-difference this [s]))
+  ([this s] (-difference this (vector s)))
   ([this s & sets] (-difference this (cons s sets))))
 
 (defn disj
   ([this] this)
   ([this x] (-disj this [x]))
-  ([this x & xs] (-disj this (cons x xs))))
+  ([this x & xs] (-disj this (clojure.core/cons x xs))))
```

Comment:
It's because we use `apply` under the hood here: https://github.com/mylesmegyesi/clojure.core/blob/master/src/clj/clojure/lang/apersistent_set.clj#L79 while we don't for `-difference`: https://github.com/mylesmegyesi/clojure.core/blob/master/src/clj/clojure/lang/apersistent_set.clj#L75